docs/framework/migration-guide/net-framework-4-migration-issues.md

```diff
@@ -315,16 +315,16 @@ Namespaces: <xref:System.Xml.Linq>; <xref:System.Xml.Schema>, <xref:System.Xml.X
 
 ### Reference
 
-[New Types and Members in the .NET Framework 4](https://msdn.microsoft.com/library/ff641764(v=vs.100).aspx)
+- [New Types and Members in the .NET Framework 4](https://msdn.microsoft.com/library/ff641764(v=vs.100).aspx)
 
 ### Concepts
 
-[Migration Guide to the .NET Framework 4](https://msdn.microsoft.com/library/ff657133(v=vs.100).aspx)   
-[What's New in the .NET Framework 4](https://msdn.microsoft.com/library/ms171868(v=vs.100).aspx)   
-[Version Compatibility in the .NET Framework](../../../docs/framework/migration-guide/version-compatibility.md)   
-[Migrating Office Solutions to the .NET Framework 4](https://msdn.microsoft.com/library/ee207231.aspx)
+- [Migration Guide to the .NET Framework 4](https://msdn.microsoft.com/library/ff657133(v=vs.100).aspx)
+- [What's New in the .NET Framework 4](https://msdn.microsoft.com/library/ms171868(v=vs.100).aspx)
+- [Version Compatibility in the .NET Framework](../../../docs/framework/migration-guide/version-compatibility.md)
+- [Migrating Office Solutions to the .NET Framework 4](https://msdn.microsoft.com/library/ee207231.aspx)
 
 ### Other resources
 
-[What's Obsolete in the .NET Framework](https://msdn.microsoft.com/library/ee461502(v=vs.110).aspx)   
-[Migration Issues for .NET Framework 4 Applications: Beta 2 to RTM](https://go.microsoft.com/fwlink/?LinkId=191505)
+- [What's Obsolete in the .NET Framework](https://msdn.microsoft.com/library/ee461502(v=vs.110).aspx)
```

Comment:
the current title is "What's Obsolete in the .NET Framework Class Library"; its URL is ../whats-new/whats-obsolete.md